tfjs-backend-webgpu/src/kernels/resize_bilinear_webgpu.ts

```diff
@@ -25,6 +25,7 @@ export class ResizeBilinearProgram implements WebGPUProgram {
   dispatchLayout: {x: number[], y: number[], z: number[]};
   dispatch: [number, number, number];
   variableNames = ['x'];
+  workGroupSize: [number, number, number] = [4, 4, 4];
```

Comment:
Try `workPerThread` > 1.

Reply:
When we use more than one dimension, we need to add a new helper function like getOutputCoordsWithWorkPerThread(int x, int y, int z). So I will do it in a separate patch.